src/DynamoCoreWpf/NodeViewCustomization/NodeViewCustomizations/FunctionNodeViewCustomization.cs

```diff
@@ -59,7 +61,6 @@ public void CustomizeView(Function function, NodeView nodeView)
 
             publishCustomNodeItem.Command = nodeView.ViewModel.DynamoViewModel.PublishSelectedNodesCommand;
             publishCustomNodeItem.CommandParameter = functionNodeModel;
-
             nodeView.UpdateLayout();
```

Comment:
is this updateLayout call useful ?
Looks like the customization only creates some context menu items. THey do not appear to be updated when the customization is applied.

Reply:
it looks like it could be deleted. But maybe we should leave it for another task

Reply:
I've also deleted this call, seems that was created by the same person that the Python node one.
https://github.com/DynamoDS/Dynamo/pull/14929/commits/a548050c048961f64876ac83d4cf2238d5791460